float64 x_angle

float64 y_angle

int8 servo_angle

int8 at_goal